web/styles/popovers.css

```diff
@@ -47,6 +47,34 @@
     }
 }
 
+.tippy-box[data-theme="navbar-dropdown-menu"] {
+    border: 0;
+    border-radius: 6px;
+
+    .tippy-content {
+        padding: 0;
+    }
+
+    .tippy-arrow {
+        top: -5.5px !important;
```

Comment:
I think we can get rid of `!important` here by using `> .tippy-arrow` to match the competing selector that sets `top: 0`.

Reply:
Yeah, done.